src/com/gmail/chernii/oleksii/forkjoinpool/ForkJoinPoolExample.java

```diff
@@ -0,0 +1,45 @@
+package com.gmail.chernii.oleksii.forkjoinpool;
+
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.RecursiveTask;
+
+/**
+ * Created by Space on 19.03.2019.
+ */
+public class ForkJoinPoolExample {
+    private static final int numberOfThread = 4;
```

Comment:
Naming is not in a java style. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names